docs/customapps.md

```diff
@@ -6,3 +6,5 @@ has_children: true
 permalink: /customapps
 ---
 # BitBoxBase: Custom applications
+
+This section outlines the use and configuration of custom applications we built for the BitBoxBase.
```

Comment:
More to come I guess, or did you leave this blank by accident?

Reply:
Ah, it's just for navigation, never mind.

Reply:
Yes, a table of content for subpages is generated automatically.